test/Microsoft.AspNetCore.Mvc.Core.Test/Infrastructure/DefaultActionSelectorTests.cs

```diff
@@ -174,6 +174,82 @@ public void SelectCandidates_NoMatch_ExcludesAttributeRoutedActions()
         }
 
         [Fact]
+        public void SelectCandidates_Match_CaseSensitiveMatch_IncludesAllCaseInsensitiveMatches()
```

Comment:
There are lots of existing tests for this feature. I'm adding tests here that will target specific details of the new implementation.

> This test isn't doing a CaseSensitiveMatch, it's using route values which are an exact match for one of the candidates 

This is what I define as a case sensitive match

> other tests uses route values that case-sensitively matches neither candidate.

This is what I defined as a case-insensitive-match